### ISSUE TITLE
add button for renew & delete ssl

### ISSUE THREAD

**Issue by alsyundawy** [Prajithp/letsencrypt-cpanel#43]
Can you add this fiture ?

**Comment by Prajithp** [comment]
Delete ssl button make sense :+1:  
We have already implemented a cronjob for renewing SSL certificate, so I don't think this is really needed.

**Comment by insxa** [comment]
Yes it needs to be added renew and delete button. And even if it renew it automaticaly, maybe for someone will not work that option

**Comment by alsyundawy** [comment]
add button too for reinstall :dango:

**Comment by itseasy21** [comment]
+1 For this request, please add a **DELETE** button

PS: Thanks for this awesome plugin :dancers:

**Comment by adminko-svk** [comment]
I think cronjob is not working - how can I check if cronjob is working?

**Comment by W5x** [comment]
+1 For this request, thank you for all your hard work.
